bin/__tests__/getIntegrationAccessToken.test.js

```diff
@@ -17,6 +17,31 @@ const METASCOPES = [
   'ent_reactor_admin_sdk',
 ];
 
+const getEnvConfig = o => {
+  return Object.assign(
```

Comment:
Not a big deal, but here you could get rid of the return statement and the brackets.